proof/refine/AARCH64/ArchInvsDefs_H.thy

```diff
@@ -40,6 +40,17 @@ context Arch begin arch_global_naming
 declare lookupPTSlotFromLevel.simps[simp del]
 declare lookupPTFromLevel.simps[simp del]
 
+(* this architecture does not use kernel_mappings / pspace_in_kernel_mappings *)
+definition kernel_mappings :: "machine_word set" where
+  "kernel_mappings \<equiv> UNIV"
+
+definition pspace_in_kernel_mappings' :: "kernel_state \<Rightarrow> bool" where
+ "pspace_in_kernel_mappings' \<equiv> \<top>"
```

Comment:
Not for now, but maybe to note in a FIXME or issue: we should really rename these two, because pspace and cap references are of course not in the kernel mappings, but in the area mapped by the kernel mappings (maybe `pspace_in_kernel_window'`, which I think we did add in RISCV64 AInvs).

For splitting, I'd leave them, though, the rename can probably be automated by script at some later point.